src/api/status/vite.config.js

```diff
@@ -0,0 +1,16 @@
+import { defineConfig } from 'vite';
+
+// vite.config.js
+export default defineConfig({
+  build: {
+    outDir: './public/dist',
+    rollupOptions: {
+      output: {
+        // Prevent hashed file names
```

Comment:
Why prevent this?

Reply:
With hashed file name (e.g. dashboard.abc123.js), the server has to read from vite manifest.json file to get the correct file name. It's simple, I did that initially but I thought that was unnecessary. Maybe it can cause a problem with browser's caching?

Reply:
It enables browser caching. When you include the hash in the filename, the browser can cache that bundle forever, since it will never change (i.e., a new build will include a new hash).  In general, you want this.